HW35_MyPet.py

```diff
@@ -0,0 +1,69 @@
+
+"""
+    Tamagotchi: can create pets that have favourite food and toy, feed them and play fetch
+
+"""
+class MyPet:
+
+    FAVOURITE_TOY = "ball"
+    FAVOURITE_FOOD = "cheese"
+    ENERGETIC = True
+    def __init__(self, animal, name):
+        self.animal = animal
+        self.name = name
+
+    def feed(self, food: str):
+        """
+        Feeds pet
+        :param food: food to feed
+        :return: pet's impression of the food
+        """
+        if food.lower() == MyPet.FAVOURITE_FOOD:
+            print("Mmmm...Yum")
+        else:
+            print("yucks....")
+
+    def play_fetch(self):
+        """
+        Pet fetches favourite toy
+        :return: makes animal tired, pet doesn't play if tired
```

Comment:
In return we should specify what data will be returned from method. Here and in methods below None will be returned. So this line as explanation should be specified just in docstring.